src/xcode/ENA/TestPlans/AllTests.xctestplan

```diff
@@ -76,6 +76,7 @@
     {
       "parallelizable" : true,
       "skippedTests" : [
+        "ELSSubmissionTests\/testELSAuthentication()",
```

Comment:
?

Reply:
This test does no longer exists. Must be an artifact 🤷‍♂️

Reply:
It's out now.